x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/outlier_exploration/outlier_exploration.tsx

```diff
@@ -82,6 +83,13 @@ export const OutlierExploration: FC<ExplorationProps> = React.memo(({ jobId }) =
 
   return (
     <EuiPanel data-test-subj="mlDFAnalyticsOutlierExplorationTablePanel">
+      {jobConfig !== undefined && needsDestIndexPattern && (
+        <IndexPatternPrompt
+          destIndex={
+            Array.isArray(jobConfig.dest?.index) ? jobConfig.dest.index[0] : jobConfig.dest.index
```

Comment:
As above, isn't `dest.index` always a string? `source.index` can be an array, but I think the dest index will always be a string.

Reply:
Same as above b7c65a8d71b075768b2c7d4434c260087de30c8a